authors/apps/articles/views.py

```diff
@@ -57,3 +61,72 @@ def destroy(self, request, pk, **kwargs):
                 return Response("You have succesfully deleted the article")
             return Response("You do not have permision to delete the article")
         return Response(self.err_message, status=status.HTTP_404_NOT_FOUND)
+
+
+class Like(APIView):
+    permission_classes = (IsAuthenticated, )
+
+    @classmethod
+    def post(self, request, **kwargs):
+        current_user = request.user.username
+        article = Article.objects.filter(id=kwargs['article_id'])
```

Comment:
You should override the default method `get_object` and query if the opbject exists.
You should also change this to something to raise a 404 incase the article doesn't exist and then return it.

```suggestion
        article = Article.get_object_or_404(id=kwargs['article_id'])
```

Reply:
I had got an error when I had tried to use this earlier. ```Article has no attribute get_object_or_404```
 That is why I am trying this other approach of getting the queryset list.
<img width="664" alt="Screen Shot 2019-05-02 at 9 04 51 AM" src="https://user-images.githubusercontent.com/44635784/57060596-b48b6400-6cc2-11e9-872d-1bbc4e42a00f.png">
<img width="534" alt="Screen Shot 2019-05-02 at 9 05 00 AM" src="https://user-images.githubusercontent.com/44635784/57060604-ba814500-6cc2-11e9-9ebf-55ee81fafe0a.png">

Reply:
@KabohaJeanMark my bad , the actual method is get_object_or_404(Model, pk=1) but we could remain with the filter since we want to return a custom message.

#### Screenshot 
<img width="872" alt="Screen Shot 2019-05-02 at 10 20 35" src="https://user-images.githubusercontent.com/36948493/57061008-20ba9780-6cc4-11e9-818b-105eae510b51.png">

Reply:
@KabohaJeanMark Always try to use the docs to find the actual implementation.

Reply:
Yes @archibishop . Thank you for this feedback. I have worked on this and checked out the docs